packages/ui/src/components/va-input/VaInput.demo.vue

```diff
@@ -269,6 +269,13 @@
         </template>
       </va-input>
     </VbCard>
+    <VbCard title="Required">
```

Comment:
Would be nice to have same demo on select, time-input etc. Just to be sure that it works here too.